Watchman.Discord/Areas/Statistics/BotCommands/StatsCommand.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Devscord.DiscordFramework.Framework.Commands;
+using Devscord.DiscordFramework.Framework.Commands.PropertyAttributes;
+using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
+
+namespace Watchman.Discord.Areas.Statistics.BotCommands
+{
+    public class StatsCommand : IBotCommand
+    {
+        [Bool]
+        public bool Hour { get; set; }
```

Comment:
Jeśli to mają być boole, to powinny nazywać się w konwencji IsHour/IsValidHour

Reply:
nie pamiętam czy przypadkiem nie jest tak, że złapie argument -hour tylko jesli property ma nazwę Hour
IsHour nie złapie, bo trzeba by pisać -stats -ishour

Reply:
nie, to jest zależne od nazwy grupy.

Reply:
albo dobra nie ważne. Zapomniałem, że to nie CustomCommand

Reply:
powinno być `Hour` bo na podstawie tego generowany jest wygląd komendy 

w przyszłości doda sie typ "Enum" do takich sytuacji
ale to dopiero w przyszłości, bo troche pracy przy tym będzie